### PR TITLE
DOC: Add GLM varfuncs and weights notebook to documentation

### PR DESCRIPTION
**Pull request by thequackdaddy**
For some reason, the `varfuncs` aren't mentioned in the docs. This changes that.

### CODE REVIEW

**Comment by thequackdaddy** [comment]
I also meant to add the GLM weights notebook to the web examples. Did that here.

File: docs/source/examples/landing.json [hunk 1]
```diff
@@ -98,6 +98,11 @@
             "target": "notebooks/generated/glm_formula.html",
             "img": "../_static/images/glm_formulas.png"
 
+        },
+        {
+            "text": "Weighting Observations with GLMs",
+            "target": "notebooks/generated/glm_weights.html",
+            "img": "../_static/images/glm_weights.png"
```
Comment (josef-pkt) [inline docs/source/examples/landing.json]: I didn't remember that we have to list notebooks here.
I think there are some notebooks like VECM that don't show up on the web doc site

Comment (thequackdaddy) [reply docs/source/examples/landing.json]: I found the following notebooks in `statsmodels/examples/notebooks` that are not in `landing.json`. 

(FYI, VECM is not one of them. I found VECM notebook gist [here](https://gist.github.com/yogabonito/b532e20b64587c19198f1aacb5ddfca2). I can copy it over if you want.)

1. chi2_fitting.ipynb
2. distributed estimation.ipynb
3. exponential_smoothing.ipynb
4. influence_glm_logit.ipynb
5. pca_fertitlity_factors.ipynb
6. plots_boxplots.ipynb
7. statespace_seasonal.ipynb

Want me to add these 7 plus the VECM notebook? Alternatively, I guess we could probably do this via `make html` in sphinx, but my guess is that's a place where the solution is far more complicated than the problem.

Comment (josef-pkt) [reply docs/source/examples/landing.json]: Ok, I found it documented here http://www.statsmodels.org/devel/dev/examples.html#the-example-gallery
I was searching issues and source a while ago, and didn't find that piece of information.

@thequackdaddy Thanks
Adding those 7 looks fine.
I need to check the VECM notebook, I ran the notebook from the gist several times with changes during review, and need to see if there are changes that need to go in. (I thought we had already added it to the docs.)

However, this PR is ready to merge, maybe after a rebase. So you could also start a new PR for adding the missing notebooks.

**Pull request merged**